lm75bd/lm75bd.c

```diff
@@ -26,9 +29,68 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
 }
 
-error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
+error_code_t readTempLM75BD(uint8_t devAddr, float *temp) { // Question 1. what do I use temp for, if theregister I am using is 0x00
   /* Implement this driver function */
-  
+    error_code_t errCode;
+
+    if (temp == NULL) return ERR_CODE_INVALID_ARG;
+
+    // to implement this we need to select the sensor's internaltemeprature register (7.4.1)
+	  uint8_t pointer_register_value = 0x00; // Point to the Temperature Register
+		devAddr = LM75BD_OBC_I2C_ADDR; 
```

Comment:
device address is passed into the function, so you don't need to set it again. You can just use the address that was passed in, this is because if we have multiple lm75bd temperature sensors each would have a different address, so we would want the address to be a parameter to choose which sensor we want to read from